Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -457,10 +462,24 @@ def retrieve_and_display_info_and_open_pr(formula_or_cask, name, repositories, a
                                       #{outdated_synced_formulae.join(", ")}.
           EOS
         end
-        puts <<~EOS unless args.no_pull_requests?
-          Duplicate pull requests:       #{duplicate_pull_requests       || "none"}
-          Maybe duplicate pull requests: #{maybe_duplicate_pull_requests || "none"}
-        EOS
+        if !args.no_pull_requests? &&
+           (new_version.general != "unable to get versions") &&
```

Comment:
Would be good to extract `unable to get versions` (and, from below, `none`) into constants.